plaidml/bridge/openvino/plaidml_ops.def

```diff
@@ -57,4 +58,4 @@ OP(Tile)
 OP(Transpose)
 OP(UnaryOps)
 OP(Unsqueeze)
-OP(VariadicSplit)
+OP(VariadicSplit)
```

Comment:
Why

Reply:
I think this is about windows using CRLF while linux using LF as the symbol of a newline.

Reply:
yes, we shall try to remove this